Vagrantfile

```diff
@@ -13,8 +26,8 @@ Vagrant.configure(2) do |config|
     config.vm.network "forwarded_port", guest: 6379, host: 6379
     config.vm.network "forwarded_port", guest: 35729, host: 35729
 
-    config.vm.synced_folder ".", "/opt/liveblog"
+    config.vm.network :private_network, ip: '192.168.50.50' # Required for NFS to work, pick any local IP
```

Comment:
Local networks don't work with LXC (as far as I can recall), so this should be inside the virtualbox block